Tests/scripts/gitlab_client.py

```diff
@@ -0,0 +1,113 @@
+import enum
+import json
+from tempfile import mkdtemp
+import zipfile
+from pathlib import Path
+from typing import Any
+
+import requests
+
+
+API_BASE_URL = "https://code.pan.run/api/v4"  # disable-secrets-detection
+PROJECT_ID = "2596"
```

Comment:
@michal-dagan Have you created a global variable for this?

Reply:
yes, please use the predefined variables:

GITLAB_SERVER_URL = get_env_var('CI_SERVER_URL', 'https://code.pan.run')  # disable-secrets-detection
GITLAB_PROJECT_ID = get_env_var('CI_PROJECT_ID', '2596')  # the default is the id of the content project in code.pan.run

GITLAB_CONTENT_PIPELINES_BASE_URL = 'http://code.pan.run/api/v4/projects/2596/pipelines/'  # disable-secrets-detection
# disable-secrets-detection
GITLAB_CONTENT_PIPELINES_BASE_URL = f'{GITLAB_SERVER_URL}/api/v4/projects/{GITLAB_PROJECT_ID}/pipelines/'